site/block-diagram/block_diagram.css

```diff
@@ -60,14 +60,21 @@
     --clock5-fg: var(--text-black);
     --clock6-fg: var(--text-black);
 
+    --tooltip-bg: #333;
+    --tooltip-fg: var(--text-white);
+    --tooltip-title-bg: #666;
+    --tooltip-title-fg: var(--text-white);
+    --tooltip-arrow-width: 0.8em;
+    --tooltip-arrow-height: 0.7em;
+
     --status1-bg: #b81d13;
     --status2-bg: #efb700;
     --status3-bg: #82be00;
     --status4-bg: #008450;
     --status1-fg: var(--text-white);
     --status2-fg: var(--text-black);
     --status3-fg: var(--text-black);
-    --status4-fg: var(--text-black);
+    --status4-fg: var(--text-white);
```

Comment:
Nit: Is this something that can be put into an earlier commit?

Reply:
Wow, good spot